helpers/constants.py

```diff
@@ -138,4 +135,12 @@
 # xref validation constants
 partial_title = "xref validation"
 xref_validation_msg = "Invalid cross references"
-
+xref_validation_module_name = "at-uploader | This doc contains a list of refs"
+# xref validation constants in CP
+Xref_linkText1 = "at-uploader | Assembly Publish test"
```

Comment:
No need to have individual constants and the again a dict containing the same values
You can access individual values from the dict itself